docs/deployment/README.md

```diff
@@ -8,6 +8,8 @@ ShowWhy is split between a front-end web-based client and a containerized Python
 * A worker node, deployed as a docker container with extra worker configuration.
 
 ## Deployment Options
-We currently deploy the application using Azure Kubernetes Services (AKS). The instructions for deploying the application using AKS are [here](./AKS_DEPLOY.md). This method uses authentication.
+We currently deploy the application using Azure Kubernetes Services (AKS). You can deploy an AKS container directly to your Azure subscription with one click, the instructions are [here](./azure-scripts/README.md). This methos does not uses authentication.
```

Comment:
"Azure Kubernetes Service"

Reply:
"This method does not use authentication"

Reply:
"\[Here are instructions\]\(link\) for deploying the application container directly into your Azure subscription.